tutor4points/core/templates/tutors.html

```diff
@@ -17,51 +17,97 @@
     <button type ="submit" name="filter" value="filter" class="filter-button" >Filter</button>
 </div> -->
 
-{% for tutor in tutors %}
-<div class="info-container d-flex" id="tutor-{{tutor.id}}" data-tutorid="{{tutor.id}}">
 
-
-    <div>
-        <img src="{{tutor.profile_pic.url}}" class="profile-pic">
+<div class="content-container">
+    <div class="alert alert-success hidden" id="success-box" role="alert">
+        {{success_message}}{{tutor_instance.first_name | title}} {{tutor_instance.last_name | title}}. Click <a href="{% url 'requests' id=user.id %}">here</a> to view all requests.
     </div>
 
-    <div class="name">
-        {{tutor.first_name | title}} {{tutor.last_name | title}} <br />
-        <div class="classes">
-            {{tutor.classes_taken}} <br />
-        </div>
-    </div>
 
+    {% for tutor in tutors %}
+    <div class="tutor-info-container d-flex" id="tutor-{{tutor.id}}" data-tutorid="{{tutor.id}}">
+        <div>
+            <img src="{{tutor.profile_pic.url}}" class="profile-pic">
 
+        </div>
 
-    <div class="times">
-        {% if tutor.times_available %}
-        {{tutor.times_available}} <br />
-        {% else %}
-        TBD <br />
-        {% endif %}
-    </div>
+        <div class="name">
+            {{tutor.first_name | title}} {{tutor.last_name | title}} <br />
+            <div class="classes">
+                {{tutor.classes_taken}} <br />
+            </div>
+        </div>
 
+        <div class="times">
+            {% if tutor.times_available %}
+            {{tutor.times_available}} <br />
+            {% else %}
+            TBD <br />
```

Comment:
It'll be helpful to write Times Available: TBD, so the user knows what the TBD is for